services/provider/proto/provider.proto

```diff
@@ -159,6 +159,10 @@ message ReportStatusRequest{
     string storageConsumerUUID = 1;
     string clientPlatformVersion = 2;
     string clientOperatorVersion = 3;
+  // clusterID is the id of the openshift cluster
+  string clusterID = 4;
+  // NamespacedName is the name and namespace of the StorageClient
+  string NamespacedName = 5;
```

Comment:
I think there is an issue with indentation here (take a look at line 161 and above for reference)